benchmarks/core/CMakeLists.txt

```diff
@@ -1,9 +1,12 @@
 # VecMem project, part of the ACTS project (R&D line)
 #
-# (c) 2021 CERN for the benefit of the ACTS project
+# (c) 2021-2022 CERN for the benefit of the ACTS project
 #
 # Mozilla Public License Version 2.0
 
+# Project include(s).
+include( vecmem-compiler-options-cpp )
```

Comment:
Not sure I fully get why this has to move but ok.

Reply:
Previously, when it was in `benchmarks/CMakeLists.txt`, the settings coming from this vecmem specific file were becoming visible to the build configuration of Google Benchmark as well. That is what I wanted to avoid. So that the external's build would be as un-modified as possible.